include/pybind11/detail/typeid.h

```diff
@@ -20,6 +20,7 @@
 
 PYBIND11_NAMESPACE_BEGIN(PYBIND11_NAMESPACE)
 PYBIND11_NAMESPACE_BEGIN(detail)
+
```

Comment:
Typo?

Reply:
I added this empty line intentionally, for consistency within this file and its easier to read. I'll keep it. Thanks!